easybuild/easyconfigs/o/OpenFOAM/external-3rd-OpenFOAM-2.3.1.patch

```diff
@@ -0,0 +1,144 @@
+#Use external CGAL and METIS (via CGAL_ROOT, BOOST_ROOT and METIS_ROOT in easyconfig)
+#B. Hajgato OCT 23,2014
```

Comment:
@boegelOwner
boegel added a note a day ago
2015?
@hajgato   
hajgato added a note just now
@boegel: The patch was made that day... Sometimes it takes time to get from develop to master...
so what should be the date at the end?